roles/postgres/vars/main.yml

```
@@ -0,0 +1,11 @@
+postgresql_version: "14"
+postgresql_daemon: "postgresql@{{ postgresql_version }}-main"
+postgres_user: "postgres"
+
+postgresql_users:
+  - name: joe
+    login_user: joe
+    role_attr_flags: SUPERUSER
```

Do we want to keep this, or set it to an empty list for now?

```suggestion
postgresql_users: []
```